CONTRIBUTING.md

```diff
@@ -36,12 +36,87 @@ library, consider developing it in a separate git repository. When it is ready t
 
+Getting Pull Requests Merged
+---
+
+There is no single person assigned to watching GitHub PR queue and ushering you through
+the process. Typically, you will ask the person that wrote a component to review
+wrote a component will review changes to it. You can find the author using `git blame`
+or asking on Discord. When working to get your PR merged, it's most important to
+understand that changing the code is your priority and not necessarily a priority
+of the person you need an approval from. Also, while you may interact the most with
+the component author, you should aim to be inclusive of others. Providing a detailed
+problem description is the most effective means of engaging both the component
+author and other potentially interested parties.
+
+### What should be in my PR description?
+
+Reviewing code is hard work and generally involves an attempt to guess the author's
+intent at various levels. Please assume reviewer time is scarce and do what you can
+to make your PR as consumable as possible. Inspired by techniques for writing good
+whitepapers, the guidance here aims to maximize reviewer engagement.
+
+Assume the reviewer will spend no more than a few seconds reading the PR title. If
+it doesn't describe a noteworthy change, don't expect the reviewer to click to see
+more.
+
+Next, like the abstract of a whitepaper, the reviewer will spend ~30 seconds reading
+the PR problem description. If what is described there doesn't look more important
+than competing issues, don't expect the reviewer to read on.
+
+Next, the reviewer will read the proposed changes. At this point, the reviewer needs
+to be convinced the proposed changes are a *good* solution to the problem described
+above.  If the proposed changes, not the code changes, generates discussion, consider
+closing the PR and returning with a design proposal instead.
+
+Finally, once the reviewer understands the problem and agrees with the approach to
+solving it, the reviewer will view the code changes. At this point, the reviewer is
+simply looking to see if the implementation actually implements what was proposed
+and if that implementation is maintainable. When a concise, readable test for each
+new code path is present, the reviewer can safely ignore the details of its
+implementation. When those tests are missing, expect to either lose engagement or
+get a pile of review comments as the reviewer attempts to consider every ambiguity
+in your implementation.
+
+### The PR Title
+
+The PR title should contain a brief summary of the change, from the perspective of the
+user. Examples of good titles:
+
+* Add rent to accounts
+* Fix out-of-memory error in validator
+* Clean up `process_message()` in runtime
+
+The conventions here are all the same as a good git commit title:
+
+* First word capitalized and in the imperative mood, not past tense ("add", not "added")
+* No trailing period
+* What was done, whom it was done to, and in what context
+
```

Comment:
Immediately after I look at the PR title, I tend to scan the state of CI on the PR with the following outcome:
1.  If CI is green continue the review
1.  If there's red I'm gone and won't be back until explicitly pinged.  I'm not going to look at your broken code.
2. If CI is pending, I will proceed with extreme caution if it's a very small PR.   Otherwise I'm gone and won't be back until explicitly pinged. 

Moral of the story -- don't bug a human until CI says you're good

Reply:
Awesome, I'll definitely add this!

Reply:
Fixed in c4ea7c0